modules/ovis/include/opencv2/ovis.hpp

```diff
@@ -105,7 +106,10 @@ class CV_EXPORTS_W WindowScene {
     CV_WRAP virtual void setEntityProperty(const String& name, int prop, const Scalar& value) = 0;
 
     /// @overload
-    CV_WRAP virtual void setEntityProperty(const String& name, int prop, const String& value) = 0;
+    CV_WRAP_AS(setEntityPropertyScale) virtual void setEntityProperty(const String& name, int prop, const String& value) = 0;
```

Comment:
needs to be done for python, so that it knows which function to call

Reply:
this should be avoided as it will break existing python code.
actually the python bindings know how to differentiate `String` and `Scalar`.
Where things fail is `int` vs. `Scalar` as the latter can be implicitly constructed from `int`.

Just remove the `int` overload and use value[0] in the Scalar variant.

Reply:
done